docs/docs/llm_as_judge.rst

```diff
@@ -0,0 +1,114 @@
+.. _llm_as_judge:
+
+.. note::
+
+   To use this tutorial, you need to :ref:`install unitxt <install_unitxt>`.
+
+=====================================
+LLM As a Judge Metrics✨
+=====================================
+
+In this section you learn how to use LLM as judge metric by unitxt. LLM as a judge is a method for evaluation the
+performance of a model based on the output of another model.
+
```

Comment:
Unitxt catalogs includes a collection of preexisting LLM as judges that can be used like any other metric.

To specify an LLM as judge metric, you can specify it in the dataset or in the recipe. For example:

card=cards.almost_evil,template=templates.qa.open.simple,"
                                          "metrics=[metrics.rag.model_response_assessment.llm_as_judge_by_flan_t5_large_on_hf_pipeline_using_mt_bench_template]",